webapp/channels/src/components/admin_console/data_grid/data_grid_row.tsx

```diff
@@ -9,12 +9,17 @@ import type {Row, Column} from './data_grid';
 
 import './data_grid.scss';
 
-type Props = {
+type DataGridRowProps = {
     columns: Column[];
     row: Row;
 }
 
-const renderCell = (row: Row, column: Column) => {
+type DataGridCellProps = {
+    column: Column;
+    row: Row;
+}
+
+const dataGridCell = ({row, column}: DataGridCellProps) => {
```

Comment:
Since this is a component, start with an Uppercase
```suggestion
const DataGridCell = ({row, column}: DataGridCellProps) => {
```

Reply:
Done